ts/defs.bzl

```diff
@@ -350,8 +350,7 @@ def ts_project(
         native.filegroup(
             name = typecheck_target_name,
             srcs = [tsc_target_name],
-            # This causes the types to be produced, which in turn triggers the tsc action to typecheck
-            output_group = "types",
+            output_group = "typecheck",
```

Comment:
This could be considered "breaking" if people are using `[name]_typecheck` to get the dts files :/

Reply:
Seems a little fringe, maybe we add it to release notes?